### PR TITLE
Wakeup improvements

### PR DESCRIPTION
**Pull request by emveepee**
After resuming from sleep Kodi will request channel update and if disconnected pvr.nextpvr was not returning an error as required by the API.

Also change to fast poll for servers after wake to account for nic startup and local server startup

### CODE REVIEW

**Comment by emveepee** [comment]
@phunkyfish when you have a chance can you please review.  I can squash them after your review if you prefer.   The Nexus backport is ready.

File: src/Channels.cpp [hunk 1]
```diff
@@ -235,6 +240,9 @@ PVR_ERROR Channels::GetChannelGroups(bool radio, kodi::addon::PVRChannelGroupsRe
       }
     }
   }
+  else {
```
Comment (phunkyfish) [inline src/Channels.cpp]: Brace on the following line

File: src/Channels.cpp [hunk 2]
```diff
@@ -160,9 +160,13 @@ PVR_ERROR Channels::GetChannels(bool radio, kodi::addon::PVRChannelsResultSet& r
       results.Add(tag);
     }
   }
-  return PVR_ERROR_NO_ERROR;
+  else {
```
Comment (phunkyfish) [inline src/Channels.cpp]: Brace on the following line

**Review by phunkyfish — APPROVED** [review]
Just some minor brace placement to fix.

File: src/Channels.cpp (hunk 1 above)
Comment (emveepee) [reply src/Channels.cpp]: Changed thanks

File: src/Channels.cpp (hunk 2 above)
Comment (emveepee) [reply src/Channels.cpp]: Also changed.

**Pull request merged**